Program.cs

```diff
@@ -18,9 +17,28 @@ class Program
 
     static void Main()
     {
-        // Array of possible words
+        bool playAgain = true;
+
+        while (playAgain)
+        {
+            PlayGame();
+
+            Console.WriteLine("\nWould you like to play again? (y/n)");
```

Comment:
Also, great that this was added so users can restart the game and keep playing without needing to manually restart the game each time.